README.md

```diff
@@ -1 +1,5 @@
 # testingspace
+
+Hello, this is an edit.
+
```

Comment:
```suggestion
### Hello, this is an edit.
```

This line would be better as H3 & no new line afterwards.

Reply:
Thanks @Steve-Mcl, really helpful.